ktor-server/ktor-server-tests/common/test/io/ktor/tests/server/routing/RoutingProcessingTest.kt

```diff
@@ -191,6 +191,28 @@ class RoutingProcessingTest {
         }
     }
 
+    @Test
+    fun testRoutingAcceptContentSubtypeWildcard() = testApplication {
+
+        routing {
+            route("getImage") {
+                accept(ContentType.Image.Any) {
+                    get { call.respondText { "Image" } }
+                }
+            }
+        }
+
+        on("making request to /getImage with Accept image/png") {
+            client.get("/getImage") {
+                header(HttpHeaders.Accept, "image/png")
+            }
+            .let {
```

Comment:
Our linter complains about indents in this file. this change should fix the problem. It also doesn't like the first empty line in the function body.

```suggestion
            }.let {
```